### DISCUSSION TITLE
Question: Distinguish programmatically triggered event from motion triggered

### DISCUSSION THREAD

**Discussion by angrycoding** [Motion-Project/motion#1499] [General]
Distinguish programmatically triggered event from motion triggered.

There is a way to programmatically trigger motion event, using webcontrol and "/action/eventstart" url. In my setup I use on_movie_end configuration option to process the movie. Now I need to determine whenever the movie was produced using manual trigger or the motion itself. Is there any way to do that without any enhancement?

**Comment by tosiara** [comment]
The code already knows if the event has been triggered by the user:
https://github.com/Motion-Project/motion/blob/master/src/webu.c#L567
But enhancement needed so that variable is available for users

**Comment by BLLuis** [comment]
I don't know if it can apply to your question, I use changed pixels value %D to detect a programmatic event start, changed pixels = 0 for programmatic event start while have a positive number for motion event start.
